content/docs/command-reference/import-url.md

```diff
@@ -337,3 +351,58 @@ $ dvc repro
 Running stage 'prepare' with command:
 	python src/prepare.py data/data.xml
 ```
+
+## Example: Import straight to the remote
+
+When you have a massive dataset in a distant location, and working on a computer
+which can't actually store it locally (due to not having enough disk space) but
+you still want to take it under control of DVC just like in the scenario of
```

Comment:
Good info. but this is repetitive (already explained in the description where this example is linked to).

Let's jump right into the example instead? Well, we can leave a 1-2 sentence intro if needed.